channel/state/signedstate_test.go

```diff
@@ -2,10 +2,11 @@ package state
 
 import (
 	"encoding/json"
+	"reflect"
 	"testing"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/google/go-cmp/cmp"
+	"github.com/go-test/deep"
 )
 
 func TestSignedStateEqual(t *testing.T) {
```

Comment:
I don't think we need this test at all, if the function it is testing is being removed.